src/components/Settings/ExtraModals/NewOllamaModel.tsx

```diff
@@ -83,12 +83,14 @@ const NewOllamaModelModal: React.FC<NewOllamaModelModalProps> = ({
         {downloadProgress && !error && (
           <p className="text-white text-sm mt-2">
             {downloadProgress.status === "success" ? (
-              "Download complete! Refrsh the chat window to use the new model."
+              "Download complete! Refresh the chat window to use the new model."
             ) : (
               <>
-                Download Progress:{" "}
-                {downloadPercentage(downloadProgress).toFixed(2)}%.
-                <p className="text-xs">(You can close this Window)</p>
+                Download progress: {downloadPercentage(downloadProgress)}
+                <p className="text-xs">
+                  (You can close this. The download will continue in the
```

Comment:
While downloading, could we disable the download button until we support multiple models? Flickering between download percentages for multiple models seems weird and it should be fairly easy to disable the button for now as a solve.

Reply:
Ended up just supporting multiple downloads